app/ui/browser-modern/actions/page-effects.js

```diff
@@ -138,3 +138,12 @@ export function captureCurrentPage() {
     dispatch(capturePage(PagesSelectors.getSelectedPageId(getState())));
   };
 }
+
+export function showDownloadNotification({ url, filename, status }) {
```

Comment:
If this is a page effect, it should be somehow related to a page. As it stands, it's a UI effect. Do we want to show these notifications for specific pages? If so, we should look into a way of doing this. Otherwise, this should be a UI effect.